draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -145,6 +145,14 @@ Since DNS resource record sets are treated atomically, a query for the verifying
 
 Additionally, placing many such TXT records at the same name increases the size of the DNS response. If the size of the response is large enough that it does not fit into a single DNS UDP packet (UDP being the most common DNS transport today), this may result in fragmentation, which often does not work reliably on the Internet today due to firewalls and middleboxes, and also is vulnerable to various attacks ([AVOID-FRAGMENTATION]). Depending on the configuration of the DNS infrastructure, it may alternatively cause the DNS server to "truncate" the UDP response and force the DNS client to re-try the query over TCP in order to get the full response. Not all networks properly transport DNS over TCP and some DNS software mistakenly believe TCP support is optional ([RFC9210]).
 
+# Scope of Validation
```

Comment:
IMO this should be in the Security Considerations section because it's something the challenge provider has to keep in mind. I don't think it has bearing on the record itself.

Reply:
I actually think Shumon's idea of calling this out is right, but perhaps Security can point back to this?

Reply:
See my previous comment Shivan. This is something the domain owner may also need to consider. So better to mention it up front.

Reply:
Gotcha, that makes sense